docs/framework/docker/aspnetmvc.md

```diff
@@ -98,17 +89,16 @@ destination folder.
 
 ## Build the image
 
-You define your Docker image in a Dockerfile that contains instructions
-for the base image, any additional components, the application you
-want to run, and other configuration image.  The Dockerfile is the input
+Define your Docker image in a Dockerfile. The Dockerfile contains instructions
+for the base image, additional components, the app you
+want to run, and other configuration images.  The Dockerfile is the input
 to the `docker build` command, which creates the image.
 
-You will build an image based on the `microsft/aspnet`
+Build an image based on the `microsft/aspnet`
```

Comment:
Again, this switch from "you'll be doing x" to "do x" (which you don't know how to do yet) sounds confusing to me